Default build does not work for 'html' module

When running a just created liftoff project and running `html:superDev` results in the exception below. 

```
Task :html:draftCompileGwt FAILED
Loading inherited module 'com.virtual_hex.games.freeuniverse.GdxDefinitionSuperdev'
   Loading inherited module 'com.virtual_hex.games.freeuniverse.GdxDefinition'
      Loading inherited module 'com.virtual_hex.games.freeuniverse.Shared'
         [ERROR] Unable to find 'com/virtual_hex/games/freeuniverse/Shared.gwt.xml' on your classpath; could be a typo, or maybe you forgot to include a classpath entry for source?
```


The resolution is to update this block
```
task addSource {
    doLast {
        sourceSets.main.compileClasspath += files(project(':core').sourceSets.main.allJava.srcDirs)
        sourceSets.main.compileClasspath += files(project(':shared').sourceSets.main.allJava.srcDirs)
    }
}
```

 in the `html` modules `build.gradle`.

--- a/src/main/kotlin/com/github/czyzby/setup/data/platforms/gwt.kt
+++ b/src/main/kotlin/com/github/czyzby/setup/data/platforms/gwt.kt
@@ -217,6 +217,7 @@ task dist(dependsOn: [clean, compileGwt]) {
 task addSource {
 	doLast {
 		sourceSets.main.compileClasspath += files(project(':core').sourceSets.main.allJava.srcDirs)
+		sourceSets.main.compileClasspath += files(project(':shared').sourceSets.main.allJava.srcDirs)
 	}
 }
 

--- a/src/main/kotlin/com/github/czyzby/setup/data/platforms/gwt.kt
+++ b/src/main/kotlin/com/github/czyzby/setup/data/platforms/gwt.kt
@@ -217,7 +217,7 @@ task dist(dependsOn: [clean, compileGwt]) {
 task addSource {
 	doLast {
 		sourceSets.main.compileClasspath += files(project(':core').sourceSets.main.allJava.srcDirs)
-		sourceSets.main.compileClasspath += files(project(':shared').sourceSets.main.allJava.srcDirs)
+		${if(project.hasPlatform(Shared.ID)) "sourceSets.main.compileClasspath += files(project(':shared').sourceSets.main.allJava.srcDirs)" else ""}
 	}
 }
 

